### DISCUSSION TITLE
Python 3.11 wheel

### DISCUSSION THREAD

**Discussion by hagsted** [sfu-db/connector-x#409] [Ideas]
Hi

I am the happy user of connectorx together with polars in python. I would like to switch to python 3.11 for the speed gains. However, the only thing that seems to hold me back is that connectorx does not have a wheel for python 3.11. What are the plans for uploading such wheel to pypi.org?

Best regards Kristian

**Comment by wangxiaoying** [comment]
Hi @hagsted , wheel files for python 3.11 are added since version `0.3.2a1`.

**Comment by fdcastel** [comment]
@wangxiaoying could you please do the same for Python 3.12?
